Gemfile

```diff
@@ -43,7 +43,7 @@ gem "jquery-ui-rails"
 gem "simple_form", '~> 3.1.0'
 gem 'nokogiri'
 gem 'equivalent-xml'
-gem 'solr_wrapper'
+gem 'solr_wrapper', '0.7.3'
```

Comment:
If it's not easy to get it working with Solr 6, then this is fine for now.  But let's make an issue to get it working with Solr 6.